__odlint.dm

```diff
@@ -0,0 +1,10 @@
+// This file is included right at the start of the DME.
+// Its purpose is to enable multiple lints (pragmas) that are supported by OpenDream to better validate the codebase
+// These are essentially nitpicks the DM compiler should pick up on but doesnt
+
+#if !defined(SPACEMAN_DMM) && defined(OPENDREAM)
```

Comment:
```suggestion
#if defined(CIBUILDING) && defined(OPENDREAM)
```

I would rather anyone building with OD only sees them as warnings and still gets to compile like with dreamchecker warnings.